test/functional/mempool_accept.py

```diff
@@ -86,27 +86,24 @@ def run_test(self):
 
         self.log.info('A transaction not in the mempool')
         fee = Decimal('0.000007')
-        raw_tx_0 = node.signrawtransactionwithwallet(node.createrawtransaction(
-            inputs=[{"txid": txid_in_block, "vout": 0, "sequence": BIP125_SEQUENCE_NUMBER}],  # RBF is used later
-            outputs=[{node.getnewaddress(): Decimal('0.3') - fee}],
-        ))['hex']
-        tx = tx_from_hex(raw_tx_0)
+        self.wallet.scan_tx(node.decoderawtransaction(raw_tx_in_block))
```

Comment:
You can use self.wallet.sendrawtransaction(node, tx), no?

Reply:
This didn't work due too high fee caused by the missing `-maxfeerate=0` parameter. Fixed that by changing the amounts now, i.e. we can use `wallet.sendrawtransaction(...)` and don't to call `wallet.scan_tx(...)` manually anymore.

Reply:
It might be cleaner to keep the amounts and change sendraw to accept and pass on **kwargs (like `def generate`).

Reply:
Nice idea, done!